applications/salk-portal/src/pages/ExperimentsPage.tsx

```diff
@@ -156,6 +157,7 @@ const ExperimentsPage = () => {
             setPopulations(getPopulations(experiment, selectedAtlas))
             dispatch(addWidget(CanvasWidget(selectedAtlas, new Set(), {})));
             dispatch(addWidget(ElectrophysiologyWidget));
+            setWidgetsReady(true)
```

Comment:
why do we need this? is it a "hack" then please add comment to this why we need this